src/main/java/uk/gov/hmcts/reform/cwrdapi/service/impl/CaseWorkerServiceImpl.java

```diff
@@ -275,11 +274,8 @@ public void publishCaseWorkerDataToTopic(List<CaseWorkerProfile> caseWorkerData)
             .collect(Collectors.toUnmodifiableList());
 
         PublishCaseWorkerData publishCaseWorkerData = new PublishCaseWorkerData();
-        ListUtils.partition(caseWorkerIds, caseWorkerDataPerMessage)
-            .forEach(data -> {
-                publishCaseWorkerData.setUserIds(data);
-                topicPublisher.sendMessage(publishCaseWorkerData);
-            });
+        publishCaseWorkerData.setUserIds(caseWorkerIds);
+        topicPublisher.sendMessage(publishCaseWorkerData);
```

Comment:
If you are not doing the partition here then you can just send `caseWorkerIds ` to `topicPublisher.sendMessage` like
`topicPublisher.sendMessage(caseWorkerIds);`